### PR TITLE
Added home btn to header

### PR DESCRIPTION
**Pull request by GavinAF**
Fixes issue: https://github.com/litmuschaos/litmus/issues/814
Signed-off-by: Gavin Foutz <gavinafoutz@gmail.com>

### CODE REVIEW

File: src/components/HomeHeader.js [hunk 1]
```diff
@@ -61,6 +61,11 @@ class HomeHeader extends React.Component {
               <h3 className="page-title">Litmus</h3>
             </div>
           </a>
+
+          <a href="https://hub.litmuschaos.io/" rel="noopener noreferrer" className="home">
```
Comment (inyee786) [inline src/components/HomeHeader.js]: use react Link tag to add tag.
and don't hardcode the link

Comment (GavinAF) [reply src/components/HomeHeader.js]: Sorry about that, never used React before.

**Comment by inyee786** [comment]
@GavinAF,
can you change base branch to `staging`

File: src/components/HomeHeader.js [hunk 2]
```diff
@@ -61,6 +61,11 @@ class HomeHeader extends React.Component {
               <h3 className="page-title">Litmus</h3>
             </div>
           </a>
+
+          <Link to={process.env.PUBLIC_URL} className="home">
+          <h3 className="top-header-home">Home</h3>
```
Comment (inyee786) [inline src/components/HomeHeader.js]: can you use home icon from here
`https://react-icons.netlify.com/#/icons/fa`

**Comment by aditya109** [comment]
@GavinAF can you please sign your PR which is the reason why DCO is failing?

**Comment by aditya109** [comment]
@GavinAF can you sign your commit? That is reason DCO is failing.

**Comment by GavinAF** [comment]
I've tried the commands listed under the DCO on how to fix it, but I must not be doing it right as it's still saying a commit is missing a sign off.

**Comment by inyee786** [comment]
@GavinAF  you can combine all the commit then push, it might work

File: src/components/HomeHeader.js (hunk 2 above)
Comment (inyee786) [reply src/components/HomeHeader.js]: @GavinAf, can please resolve this comment

Comment (GavinAF) [reply src/components/HomeHeader.js]: @inyee786, you didn't specify if you wanted it added to the left or replace, so I replaced it. Let me know if you want it the other way.

**Comment by NisantaSahoo** [comment]
Thanks for taking part in Hacktoberfest 2019. Please fill up this form to claim your swag https://hubs.ly/H0l5ldm0

**Pull request merged**